addons/website/static/src/snippets/s_countdown/options.js

```diff
@@ -33,6 +42,9 @@ options.registry.countdown = options.Class.extend({
                 this.endMessage = $message[0].outerHTML;
             }
         }
+        if (widgetValue === 'redirect' && !this.$target[0].dataset.redirectUrl) {
```

Comment:
wrong fix, this leaves [dead code](https://github.com/odoo/odoo/blame/464968de36df259ddf03e56e259f7fb17a2dc5b9/addons/website/views/snippets/s_countdown.xml#L32) and add another layer of code on top to make it work.
This should already works with default value that should be used as placeholder.
(note that furthermore we don't want to add default value here, better to not redirect than redirecting to a 404)

@qsm-odoo Was the default value used as placeholder removed/broken on purpose? Indeed between my s_countdown commit and now the behavior got broken, just had a quick look

Reply:
@rdeodoo no, just a side effect of trying to make you happy :smile:
- Before: the default value appeared as a placeholder so even though the URL was not forced on the $target, the input was empty (like in master), so the placeholder is visible (tricking the user into thinking it is actually used on the $target) -> hidden bug
- Now: same as before, just that there is no placeholder -> revealed "bug"

So I think @fja-odoo fix is the right one since it does what you wanted to do. If you think that is it better to not have an URL at all, then the fix is just about removing the default value of the original implementation. Probably better I guess.

Reply:
I think the behavior should be to have a placeholder value by default, as it is the case, but no hidden feature that adds the default value as URL that will lead to 404 pages at the countdown end, not really great.

And to not prevent the input to actually be cleared, if possible.

Like a normal input which has a placeholder I guess?

At the same time preventing the user to have no value in redirect seems cool in UX, even if 404 will be worst, so not sure..

I guess it's fine that way with the current fix then, you might want to ask "des gens compétent".

Reply:
Add data-placeholder in UserValueInput ? Would simply set the value as placeholder attr
It is true that having a default url that is a 404 and has no real purpose is useless.
We also can't really add a generic landing page like`/contactus-thank-you`.

Reply:
@fja-odoo Remove the default value and add `placeholder="e.g. /blablabla"` (not data-placeholder)

Reply:
Looks like it is` data-placeholder` that is used as placeholder in `InputUserValueWidget`

Reply:
Ok, then it is a stupid mistake I made, can you change so that it is placeholder? Maybe in 13.2. Otherwise it is not translatable.